### PR TITLE
Add telemetry docs

### PR DESCRIPTION
**Pull request by disq**
[Code PR](https://github.com/cloudquery/cloudquery/pull/280)
[RFC](https://www.notion.so/cloudquery/Analytics-RFC-1a7e777b36394c229af28d58d8fdef8b)

Try it: https://e30c541a.docs-444.pages.dev/docs/cli/telemetry
Vercel: https://docs-git-feat-telemetry-docs-cloudquery.vercel.app/docs/cli/telemetry

Haven't put it in the sidebar as I don't feel like it's a major talking point, should be under some other page probably.

### CODE REVIEW

File: docs/telemetry.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- We don't store any credentials
+- We don't store any logs
+
+## Inspecting Telemetry Contents
+
+To check what kind of data is collected, invoke `cloudquery` executable with the `--inspect-telemetry` option. This will stop sending telemetry information and write it to a local file in the current directory, `cq-telemetry.txt`. You can then inspect file contents and decide whether to opt-out or not.
+
+```bash
+# Invoke CloudQuery
+cloudquery --inspect-telemetry [operation] [arguments]
```
Comment (ukmadlz) [inline docs/telemetry.md:41]: Should this flag be added to `/docs/cli/commands/options`?

Comment (ukmadlz) [reply docs/telemetry.md:41]: Same with the below `--no-telemtry`

**Review by ukmadlz — COMMENTED** [review]
Where in the side navigation should this live?

File: docs/telemetry.md (hunk 1 above)
Comment (disq) [reply docs/telemetry.md:41]: Added

**Comment by disq** [comment]
Last commit separates the options into 3 parts... could revert if it's too much for now

**Review by ukmadlz — APPROVED** [review]

**Review by yevgenypats — APPROVED** [review]
Awesome docs! ❤️

**Pull request merged**